src/zjs_callbacks.c

```diff
@@ -424,35 +414,39 @@ void zjs_call_callback(int32_t id, void* data, uint32_t sz)
 void zjs_service_callbacks(void)
 {
     if (ring_buf_initialized) {
-        uint16_t type;
-        uint8_t value;
-        uint8_t size = 0;
-        uint32_t dummy;
-        // setting size = 0 will check if there is an item in the ring buffer
-        while (zjs_port_ring_buf_get(&ring_buffer,
-                                     &type,
-                                     &value,
-                                     &dummy,
-                                     &size) == -EMSGSIZE) {
-            if (size == 0) {
-                break;
-            } else {
+        while (1) {
```

Comment:
Yeah, I think that would work. And I think it should be at least 64 because you can fit that many callbacks in your 1024-byte buffer even if each takes up 16 bytes. I thought I remembered you'd added this in a follow-on patch but I don't see it now and am confused. :)